torch/csrc/jit/mobile/interpreter.cpp

```diff
@@ -57,6 +57,11 @@ bool InterpreterState::run(Stack& stack) {
       auto inst_with_handle = code_->instructions_with_handles_.at(pc);
       Instruction inst = inst_with_handle.instruction;
       DebugHandle debug_handle = inst_with_handle.debug_handle;
+      // If no valid debug handle found then just log pc.
+      // This is possible when we did not save debug handles
```

Comment:
This comment is a bit confusing.
It makes it sound like it was possible to do it but not anymore.
The comment also sounds like we're logging pc but that things will fail?

If logging pc's is going to cause some failure, and/or not very useful, should we instead disable logging altogether?

Reply:
I should remove this part "      // In that case post processing callback in EdgeProfile will fail
      // to retrive usefule information.
"
Since this PR adds `hasDebugHandles` API, if debug handles were present, then what is logged in events is debug handle, otherwise it is pc.

Now why this diverging behavior:
This is mainly to provide some unique identifier to the events, in the absence of debug_handles. Otherwise, for the purpose of generating summary from events that we use in benchmarking, two different conv ops will be lumped together as same event.

This behavior is possible because saving debug info is still optional. Once it is not, I think we can guarantee that debug handles are always present, but again user can explicitly chose to note save those also.